syz-manager/manager.go

```diff
@@ -584,6 +587,10 @@ func (mgr *Manager) runInstance(index int) (*Crash, error) {
 		log.Logf(0, "vm-%v: running for %v, restarting", index, time.Since(start))
 		return nil, nil
 	}
+
+	if info, ok := mgr.machineInfo[index]; ok {
+		rep.Report = append(rep.Report, info...)
```

Comment:
Would it be better to make a new MachineInfo field (or a MachineInfo struct field) on the report.Report type, rather than combining the machine info with the text of the crash?

Reply:
Yes, this should not go into Report. Report is embed into each email. And Report is what produced by pkg/report, this is unrelated. The Crash object below is a good candidate to store it for later.